simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/support/JsonMessageTypeFactory.java

```diff
@@ -53,10 +54,16 @@ public Call withPayload(Object payload) {
             return this;
         }
 
-        public String toJson() throws JsonProcessingException {
-            String payloadJson = JSON_OBJECT_MAPPER.writeValueAsString(payload);
+        public String toJson() {
+
+            try {
+                String payloadJson = JSON_OBJECT_MAPPER.writeValueAsString(payload);
+
+                return "[2,\"" + messageId + "\",\"" + action + "\"," + payloadJson + "]";
```

Comment:
Can we use String.Format just for readability here (and other places)?

Reply:
done